utils/clixon/Makefile

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (C) 2020-2023 Olof Hagsand and Rubicon Communications, LLC(Netgate)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=clixon
+PKG_VERSION:=6.5.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Olof Hagsand <olof@hagsand.se>, Philip Prindeville <philipp@redfish-solutions.com>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/clicon/$(PKG_NAME)/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=c85bf3112ddd9dcc00965780c21bf1589095c8b67f741ef7059c805feccf3bfc
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILE:=LICENSE.md
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/clixon
+  SECTION:=utils
+  CATEGORY:=Utilities
+  URL:=http://www.clixon.org
+  TITLE:=YANG-based toolchain
+  DEPENDS:=+cligen +libopenssl +libnghttp2 +libcurl \
+	 @(PACKAGE_openssh-server||PACKAGE_openssh-server-pam)
+  USERID:=clicon=61:clicon=61
+endef
+
+define Package/clixon/description
+  YANG-based toolchain including NETCONF and RESTCONF interfaces and an interactive CLI
+endef
+
+define Package/clixon/conffiles
+  /etc/clixon/clixon.xml
+  /etc/ssh/sshd_config.d/90-netconf-subsystem.conf
```

Comment:
Why there is indentation here? Did you verify that it indeed works? Because there shoukld be no indentation at all

Reply:
Fixed